### ISSUE TITLE
Miscellaneous search ranking issues

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/morphodict#1108]
This is an issue for collecting possible glitches in search ranking, which may be due to an error in the formula or the supporting FSTs, or something else:
- [ ] Seaching _my bears_ `bear +N+Px1Sg+Pl` receives zero POS-match for _maskwa_, though one should receive at least 0.333, cf.
<img width="247" alt="image" src="https://user-images.githubusercontent.com/15003407/174891050-c1511983-040b-44ed-a009-785fac6e8111.png"> <img width="409" alt="image" src="https://user-images.githubusercontent.com/15003407/174890980-1c6ca1bb-f1a6-4fe7-a45e-3eac2df3afe7.png">
- [ ] _my bear_ does not receive the POS code `+N` (this is a bug in the phrase analyzer FST).
<img width="227" alt="image" src="https://user-images.githubusercontent.com/15003407/174891609-46d555f1-a3b2-448b-ada8-93f4ea40c7f1.png">

- [ ] missing cosine vector distance value and/or morpheme ranking value:
<img width="477" alt="image" src="https://user-images.githubusercontent.com/15003407/175210654-902b06ee-d4c7-4828-84a3-6874325bfc49.png">

**Comment by fbanados** [comment]
Addition:  `nîpin` now appears on CW both as VII and NI, in two entries.  If this is correct, there's a question of whether noun or verb should get priority, all other things being equal.

**Comment by aarppe** [comment]
Yes, there are a few words that are used both as nouns and verbs, usually expressions of time, such as for day, cf. https://itwewina.altlab.app/search?q=k%C3%AEsik%C3%A2w

For ranking in search, this should be based on whatever comes out of the ranking algorithm, based on the various frequencies of lemmas, morphemes, and glossary occurrence (as those should distinguish between noun and verb readings).
